README.md

```diff
@@ -5,48 +5,41 @@ A minimal and lightweight front-end for dawn.
 Dusk translates a subset of Python code (a Python embedded domain specific language - eDSL) to SIR. Its purpose is to allow quick
 prototyping of dawn's unstructured features for internal experiments.
 
-Dusk is currently in an experimental state. An example is available here: [test.py](test.py)
+Dusk is currently in an experimental state. Examples are available here: [tests/stencils](tests/stencils/)
 
 # Requirements
 
   * Python 3.8 (support for other Python versions is planned)
   * dawn4py (see: https://github.com/MeteoSwiss-APN/dawn/tree/master/dawn/src/dawn4py )
-  
-# Usage
 
-Due to it's prototyping nature, `PYTHONPATH` has to be used for execution. For example:
+# Installation
+
+It is highly recommended to use a virtual env for development:
 
 ```bash
-PYTHONPATH=./:$PYTHONPATH ./dusk/cli.py ./test.py
+python -m venv virtual_env # create virtual env
+source virtual_env/bin/activate # activate virtual env
 ```
 
-Usage:
+Then dusk can be directly installed from github via pip (this will also install dawn4py):
 
 ```bash
-PYTHONPATH=./:$PYTHONPATH ./dusk/cli.py --help
-usage: cli.py [-h] [-o OUT_FILE] [--dump-sir] [-b {ico-cuda,ico-naive}] in_file
+pip install dusk@git+https://github.com/dawn-ico/dusk.git
+```
 
-Transforms the Python embedded DSL to SIR.
+# Usage
 
-positional arguments:
-  in_file
+The package will install a `dusk` command-line tool which can be used to compile dusk stencils:
 
-optional arguments:
-  -h, --help            show this help message and exit
-  -o OUT_FILE
-  --dump-sir            dump sir to disk
-  -b {ico-cuda,ico-naive}
+```bash
+dusk --help
 ```
 
-# Development
-
-Please use the [black](https://github.com/psf/black) formater for your python code.
-
 ## Overview
-- [test.py](test.py) - An example of the dusk eDSL
-- [dusk/script.py](dusk/script.py) - Contains definitions & mocks for the DSL
-- [dusk/cli.py](dusk/cli.py) - Implements a basic command line interface to compile the dusk eDSL
-- [dusk/transpile.py](dusk/transpile.py) - Provides a programmatic interface to compile the dusk eDSL
+- [tests/stencils](tests/stencils) - Examples of the dusk eDSL
+- [dusk/script/__init__.py](dusk/script/__init__.py) - Contains definitions & mocks for dusk
```

Comment:
```suggestion
- [dusk/script/\_\_init\_\_.py](dusk/script/__init__.py) - Contains definitions & mocks for dusk
```
Need to escape underscore so that markdown doesn't use it for formatting.